#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_add_fpi_for_htmltblx_mod_to_43_catalogs.dpatch by Adam Di Carlo <aph@debian.org>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.3/{catalog.xml,docbook.cat}: The catalogs for DB 4.3 miss
## DP:   the FPI "-//OASIS//ELEMENTS DocBook XML HTML Tables V4.3//EN".
## DP:
## DP: <URL:http://bugs.debian.org/268364>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.4-1>

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.3/catalog.xml docbook-xml-4.5/docbook-4.3/catalog.xml
--- docbook-xml-4.5~/docbook-4.3/catalog.xml	2007-03-25 23:29:54.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/catalog.xml	2007-03-25 23:31:04.000000000 +0200
@@ -40,6 +40,9 @@
 <public publicId="-//OASIS//DTD DocBook CALS Table Model V4.3//EN"
         uri="calstblx.dtd"/>
 
+<public publicId="-//OASIS//ELEMENTS DocBook XML HTML Tables V4.3//EN"
+        uri="htmltblx.mod"/>
+
 <public publicId="-//OASIS//DTD XML Exchange Table Model 19990315//EN"
         uri="soextblx.dtd"/>
 
diff -urNad docbook-xml-4.5~/docbook-4.3/docbook.cat docbook-xml-4.5/docbook-4.3/docbook.cat
--- docbook-xml-4.5~/docbook-4.3/docbook.cat	2007-03-25 23:29:53.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/docbook.cat	2007-03-25 23:31:42.000000000 +0200
@@ -33,6 +33,9 @@
 PUBLIC "-//OASIS//DTD DocBook CALS Table Model V4.3//EN"
        "calstblx.dtd"
 
+PUBLIC "-//OASIS//ELEMENTS DocBook XML HTML Tables V4.3//EN"
+       "htmltblx.mod"
+
 PUBLIC "-//OASIS//DTD XML Exchange Table Model 19990315//EN"
        "soextblx.dtd"
 
